<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoUI</class>
 <widget class="QMainWindow" name="VideoUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#panel {
	background-color: rgba(255, 255, 255, 120);
	border-radius: 10px;
}

QPushButton {
	background-color: transparent;
	border: 1px solid grey;
	border-radius: 5px;
}

QPushButton::pressed {
	background-color: rgba(0,0,0,50);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="panel" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>800</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <widget class="QLabel" name="progressLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>17</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>00:00/00:00</string>
     </property>
    </widget>
    <widget class="QSlider" name="progressBar">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>691</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="ThemedButton" name="buttonPreviousFrame">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="uires.qrc">
       <normaloff>:/ui/back.png</normaloff>:/ui/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
    </widget>
    <widget class="ThemedButton" name="buttonBackward">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Backward</string>
     </property>
    </widget>
    <widget class="ThemedButton" name="buttonPlay">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="uires.qrc">
       <normaloff>:/ui/play.png</normaloff>:/ui/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="ThemedButton" name="buttonForward">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
    <widget class="ThemedButton" name="buttonNextFrame">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="uires.qrc">
       <normaloff>:/ui/forward.png</normaloff>:/ui/forward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
    </widget>
    <widget class="ThemedButton" name="buttonQuit">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="VolumeButton" name="buttonVolume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="uires.qrc">
       <normaloff>:/ui/vol/highvolume.png</normaloff>:/ui/vol/highvolume.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="volumeBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>331</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>panel</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThemedButton</class>
   <extends>QPushButton</extends>
   <header>themedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>VolumeButton</class>
   <extends>QPushButton</extends>
   <header>volumebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="uires.qrc"/>
 </resources>
 <connections/>
</ui>
